e2e/events/input/deploy.nomad

```diff
@@ -0,0 +1,39 @@
+job "deployment_auto.nomad" {
+  datacenters = ["dc1"]
```

Comment:
The test targets for nightly are spread across multiple DCs for constraints testing. If you don't care about that for your test, it's a good idea to use both of them to give you enough nodes to spread out.

```suggestion
  datacenters = ["dc1", "dc2"]
```